loopfib.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+/*print fibonacci numbers with loop
+this one doesn't quite work and just prints out 23 when you type 4, I'm not quite sure why...
+  @author Aly Milich
+  */
+
+int main(){
+	int input;
+	int fibonacci[input];
```

Comment:
After this fix, it works for the most part. The first numbers aren't printed out when needed though. Add comments.